lib/dragonfly_libvips/processors/thumb.rb

```diff
@@ -33,7 +33,7 @@ def call(content, geometry, options = {})
               thumbnail_options[:yshrink] = dimensions.y_scale
               img.shrink(dimensions.x_scale, dimensions.y_scale, **thumbnail_options)
             when :crop
-              thumbnail_options.except!(:height, :size)
+              thumbnail_options.except!(:height, :size, :no_rotate)
```

Comment:
Option `:no_rotate` is passed to the crop method in my app and breaks with an unknown option error. Based on [ruby-vips docs](https://www.rubydoc.info/gems/ruby-vips/Vips/Image#crop-instance_method) passing `thumbnail_options` might not be needed here at all.